cfgov/preprocessed/css/video-player.less

```diff
@@ -148,7 +148,7 @@
 
 
 // Hero Video Player
-
+/*
```

Comment:
I think we can delete it, it'll be moved into the Featured Content Module later.